Makefile

```diff
@@ -7,7 +7,7 @@ DIRS := $(shell find $(SRC) -type d)
 SRCS := $(shell find $(SRC) -type f -name "*.c")
 OBJS := $(SRCS:%.c=$(OBJDIR)/%.o)
 
-CFLAGS = -Iinclude -Wall -Wpedantic -Wextra -Wshadow -std=c11
+CFLAGS = -Iinclude -Wall -Wpedantic -Wextra -Wshadow -std=gnu11
```

Comment:
I would prefer if we could compile according to the c11 standard. This works for me. I will look into what the problems may be.

Reply:
"-std=gnu11 (for C11 with GNU extensions)."